.gitignore

```diff
@@ -3,3 +3,6 @@ dist/
 node_modules/
 npm-debug.log
 .env
+.vscode/
```

Comment:
.vscode is a local user related file. please don't remove this. If you wish to ignore `.vscode/`, it should be added to your personal gitignore